x-pack/test/functional/apps/machine_learning/pages.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaFunctionalTestDefaultProviders } from '../../../types/providers';
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: KibanaFunctionalTestDefaultProviders) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  // eslint-disable-next-line ban/ban
+  describe.only('page navigation', () => {
```

Comment:
Probably want to avoid disabling this eslint check so that these don't slip through!